test/handlers_test.go

```diff
@@ -17,8 +17,10 @@ func Test_ValidateDeployRequest_ValidCharacters(t *testing.T) {
 	}{
 		{"lower", "abz"},
 		{"upper", "ABZ"},
-		{"upper and lower mixed", "AbZ"},
-		{"includes dashes", "test-function"},
+		{"upper and lower mixed", "abz"},
```

Comment:
This is an invalid test scenario name.. it's no longer upper and lower mixed. This should be moved into the "invalid" names test.